config/passport.js

```diff
@@ -1 +1,29 @@
-import passport from "./node_modules/passport";
```

Comment:
When importing a an installed module, you don't need to necessary include node_modules

Reply:
that line of code was already there in the previous original source code, am padoning him on that, but atleast he should have removed it